REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.4
 DataStructures
 Combinatorics
+Compat 0.17
```

Comment:
the new style call overloading is older than this - can look it up exactly later